src/wasm/WasmModule.cpp

```diff
@@ -36,6 +46,12 @@ size_t getNumberOfWasmPagesForBytes(uint32_t nBytes)
     return pageCount;
 }
 
+uint32_t roundUpToWasmPageAligned(uint32_t nBytes)
+{
+    size_t nPages = getNumberOfWasmPagesForBytes(nBytes);
+    return (uint32_t)(nPages * WASM_BYTES_PER_PAGE);
```

Comment:
Could be more efficient but it's readable and only used in a couple of places atm.